saltedge-compliance-connector-java-v2/compliance-connector-sdk/src/main/java/com/saltedge/connector/sdk/provider/ConnectorSDKCallbackService.java

```diff
@@ -194,6 +195,17 @@ public String onPaymentInitiationAuthorizationSuccess(
     return paymentExtraMap.getOrDefault(SDKConstants.KEY_RETURN_TO_URL, "");
   }
 
+  @Override
+  public boolean updatePaymentFundsInformation(Boolean fundsAvailable, String paymentExtra, String paymentProduct) {
+    Map<String, String> paymentExtraMap = parseExtra(paymentExtra);
+    String sessionSecret = paymentExtraMap.get(SDKConstants.KEY_SESSION_SECRET);
+    String status = getStatusOfPaymentProduct(paymentProduct);
```

Comment:
rename `getStatusOfPaymentProduct` to `getFinalStatusOfPaymentProduct`.  
On intermediate stages we should not calculate status.